src/main/java/com/adaptavist/tm4j/jenkins/extensions/configuration/Tm4jGlobalConfiguration.java

```diff
@@ -46,47 +48,67 @@ public boolean configure(StaplerRequest request, JSONObject formData) throws For
         Permissions.checkAdminPermission();
         request.bindParameters(this);
         Object formJiraInstances = formData.get("jiraInstances");
+        JSONArray jiraInstancesList = new JSONArray();
+        if(formJiraInstances instanceof JSONArray){
```

Comment:
formatting

Reply:
we need to add check style  in this project